tests/unit/Lance.spec.js

```diff
@@ -0,0 +1,46 @@
+import Lance from '@/components/Lance'
+import { mount } from '@vue/test-utils'
+
+describe('Lance', () => {
+  test('deveria dar o primeiro lance maior do que 0', () => {
+    const wrapper = mount(Lance)
+    const input = wrapper.find('input')
+    input.setValue(100)
+    wrapper.trigger('submit')
+    const lancesEmitidos = wrapper.emitted('novo-lance')
+    expect(lancesEmitidos).toHaveLength(1)
+    const lance = parseInt(wrapper.emitted('novo-lance')[0][0])
+    expect(lance).toBe(100)
+  })
```

Comment:
Vc fez o teste do caminho feliz, mas faltou o teste de se o valor for menor ou igual a 0